zproject/dev_settings.py

```diff
@@ -209,7 +209,7 @@
 PUSH_NOTIFICATION_BOUNCER_URL = f"http://push.{EXTERNAL_HOST}"
 
 # Breaks the UI if used, but enabled for development environment testing.
-ALLOW_ANONYMOUS_GROUP_VALUED_SETTINGS = True
+ALLOW_GROUP_VALUED_SETTINGS = True
```

Comment:
Renamed this setting as we discussed previously in https://chat.zulip.org/#narrow/stream/378-api-design/topic/managing.20resolve.20topic.20permissions.20independently/near/1810442.